spec/spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
You should move `SimpleCov` into `rails_helper` and call `SimpleCov.start 'rails'` to ensure you'd not over inflating your testing.  We wouldn't want that, would we? :wink: